tests/contrib/requests/test_requests.py

```diff
@@ -270,6 +270,32 @@ def test_split_by_domain_wrong(self):
 
         eq_(s.service, 'requests')
 
+    def test_split_by_domain_remove_auth_in_url(self):
+        # ensure that auth details are stripped from URL
+        cfg = config.get_from(self.session)
+        cfg['split_by_domain'] = True
+        out = self.session.get('http://user:pass@httpbin.org')
+        eq_(out.status_code, 200)
+
+        spans = self.tracer.writer.pop()
+        eq_(len(spans), 1)
+        s = spans[0]
+
+        eq_(s.service, 'httpbin.org')
+
+    def test_split_by_domain_includes_port(self):
+        # ensure that port is included if present in URL
+        cfg = config.get_from(self.session)
+        cfg['split_by_domain'] = True
+        out = self.session.get('https://httpbin.org:443')
```

Comment:
`httpbin.org` doesn't support HTTPS 🙄 

```suggestion
        out = self.session.get('http://httpbin.org:80')
```


☝️ also, first time trying this beta feature... kind of cool!

Reply:
hmm, worked locally but I guess I wasn't running the docker service